Makefile

```diff
@@ -163,8 +163,7 @@ verify:
 	hack/verify-vendor-licenses.sh
 
 lint: ## Lint the files
-	golangci-lint version
-	golangci-lint run -v
+	hack/verify-golangci-lint.sh
```

Comment:
Have you tested this Makefile on your machine?

Reply:
yes